src/Octoshift/GithubClient.cs

```diff
@@ -186,8 +204,24 @@ private string GetNextUrl(KeyValuePair<string, IEnumerable<string>>[] headers)
         private string ExtractLinkHeader(IEnumerable<KeyValuePair<string, IEnumerable<string>>> headers) =>
             ExtractHeaderValue("Link", headers);
 
+        private int RateLimitRemaining(IEnumerable<KeyValuePair<string, IEnumerable<string>>> headers) =>
+            int.Parse(ExtractHeaderValue("X-RateLimit-Remaining", headers) ?? "5000");
+
+        private long RateLimitReset(IEnumerable<KeyValuePair<string, IEnumerable<string>>> headers) =>
+            long.Parse(ExtractHeaderValue("X-RateLimit-Reset", headers) ?? DateTimeOffset.Now.ToUnixTimeSeconds().ToString());
+
```

Comment:
It's customary in C# for method names to be verbs or verb phrases. Microsoft also recommends it in [design guidelines for naming methods](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members#names-of-methods):
> Because methods are the means of taking action, the design guidelines require that method names be verbs or verb phrases. Following this guideline also serves to distinguish method names from property and type names, which are noun or adjective phrases.

So these methods are better to be named as `GetRemainingRateLimit` and `ResetRateLimit`.

Reply:
Should probably be GetRateLimitReset based on my understanding of what it's doing.

Reply:
Oh yeah I though it was trying to reset the rate limit (this is exactly why we should rename it 😃), but didn't notice that it was actually returning the rate limit reset 😄